crates/bindings-csharp/Runtime/Attrs.cs

```diff
@@ -51,6 +51,17 @@ public sealed class TableAttribute : Attribute
         public string? Scheduled { get; init; }
     }
 
+    public static class Index {
+        [AttributeUsage(AttributeTargets.Struct | AttributeTargets.Class, AllowMultiple = true)]
+        public sealed class BTreeAttribute : Attribute {
+            public string? Table { get; init; }
+
+            public string? Name { get; init; }
+
+            public string[] Columns { get; init; } = [];
```

Comment:
```suggestion
            public required string[] Columns { get; init; };
```

IIUC, index without columns is never useful, so let's use the neat `required` feature to make sure that user always specifies columns.